feature-libs/order/root/order-root.module.ts

```diff
@@ -62,56 +62,56 @@ export function defaultCartOrderComponentsConfig(): CmsConfig {
         data: { cxRoute: 'orderDetails' },
       },
       {
+        // @ts-ignore
```

Comment:
I presume the `// @ts-ignore` couldn't be removed on the 2 first items.

Reply:
`// @ts-ignore` ignores this `path: null,`